### PR TITLE
Move compatibilityNotice showcase to astro

### PR DESCRIPTION
**Pull request by lmuntaner**
This migrates the `compatibilityNotice` page to a static Astro page to avoid having it in the hardcoded list of showcase components.

### CODE REVIEW

**Comment by lmuntaner** [comment]
Current dependencies on/for this PR:
* **#2307** <a href="https://app.graphite.dev/github/pr/dfinity/internet-identity/2307?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/dfinity/internet-identity/2307?utm_source=stack-comment).

**Comment by lmuntaner** [comment]
@nmattia please review

File: src/showcase/src/layouts/Screen.astro [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+import Layout from "../layouts/Layout.astro";
+
+const { title, pageName } = Astro.props;
```
Comment (nmattia) [inline src/showcase/src/layouts/Screen.astro:4]: do you know if Astro types the props if you don't specify the types? or does is default to `any`?

Comment (lmuntaner) [reply src/showcase/src/layouts/Screen.astro:12]: Good point! Yes, I found it here: https://docs.astro.build/en/guides/typescript/#component-props

Done

File: src/showcase/src/layouts/Screen.astro [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+import Layout from "../layouts/Layout.astro";
+
+interface Props {
```
Comment (nmattia) [inline src/showcase/src/layouts/Screen.astro:4]: nit: we use `type` pretty much everywhere in the codebase

File: src/showcase/src/layouts/Screen.astro (hunk 1 above)
Comment (nmattia) [reply src/showcase/src/layouts/Screen.astro:12]: Thanks! As far as I'm concerned you can even drop the `children` field

**Review by nmattia — APPROVED** [review]

File: src/showcase/src/layouts/Screen.astro (hunk 2 above)
Comment (lmuntaner) [reply src/showcase/src/layouts/Screen.astro:4]: Sounds good

File: src/showcase/src/layouts/Screen.astro (hunk 1 above)
Comment (lmuntaner) [reply src/showcase/src/layouts/Screen.astro:12]: The recommendation is that if the component requires a "slot" we should add it in the props.

This way I got an error when I used `<Screen>` without a child.

**Pull request merged**